src/Header.css

```diff
@@ -0,0 +1,37 @@
+.Header {
+  text-align: center;
+}
+
+.Header-logo {
+  animation: App-logo-spin infinite 20s linear;
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.App-header {
```

Comment:
Is this used in Header.js? make sure you don't have repeated CSS from parent components when copying files